src/deadline/client/ui/dev_application.py

```diff
@@ -95,7 +95,7 @@ def app() -> None:
     app.setStyle(QStyleFactory.create("fusion"))
 
     # Set the application info.
-    app.setApplicationName("AWS Deadline Cloud Client Test GUI")
+    app.setApplicationName("AWS Deadline Cloud client test gui")
```

Comment:
GUI could stay capitalized, since it's an acronym.

Reply:
updated!